cylc/flow/simulation.py

```diff
@@ -166,46 +200,63 @@ def sim_time_check(
     Returns:
         True if _any_ simulated task state has changed.
     """
-    sim_task_state_changed = False
-    now = time()
+    sim_task_state_changed: bool = False
     for itask in itasks:
-        if itask.state.status != TASK_STATUS_RUNNING:
+        if (
+            itask.state.status != TASK_STATUS_RUNNING
+            or any([
+                itask.state.is_queued,
+                itask.state.is_held,
+                itask.state.is_runahead
+            ])
```

Comment:
No real gain to using `any()` here, and disadvantage of a performance penalty
```suggestion
            or itask.state.is_queued
            or itask.state.is_held
            or itask.state.is_runahead
```